Track Your Reps/Views/RecentEventsVC.swift

```diff
@@ -0,0 +1,42 @@
+///
```

Comment:
do you have a working example, because I would like to do it this way in a refactor